Client/Frontend/Onboarding/Models/IntroViewModel.swift

```diff
@@ -44,4 +58,15 @@ class IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
     func saveHasSeenOnboarding() {
         introScreenManager?.didSeeIntroScreen()
     }
+
+    // MARK: SkAdNetwork
+    // this event should be sent in the first 24h time window, if it's not sent the conversion value is locked by Apple
+    func sendOnboardingUserActivationEvent() {
+        let fineValue = OnboardingOptions.allCases.map { chosenOptions.contains($0) ? $0.rawValue : 0 }.reduce(0, +)
```

Comment:
nit: iterating through the selected options may be easier to understand and still correct
```suggestion
        let fineValue = chosenOptions.map { $0.rawValue }.reduce(0, +)
```

Reply:
that's how I would have done it in normal circumstances but here we are using OptionSet for the bitmask which does not conform to the 'Sequence' protocol so it can't be iterated through, nor does it support map or reduce and making the OptionSet conform to Sequence can be done but it would make the code more unreadable than making it conform to CaseIterable

Reply:
Got it, clearly I'm still learning about swift! thank you!